LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016
+Copyright (c) 2017
```

Comment:
I actually manually made that change, not ember-cli's fault! :) I'll change it to your suggestion.